apps/server/src/infra/tldraw-client/tldraw-client.adapter.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@nestjs/common';
+import { TldrawDocumentApi } from './generated';
+
+@Injectable()
+export class TldrawClientAdapter {
+	constructor(private readonly tldrawDocumentApi: TldrawDocumentApi) {}
+
+	async deleteDrawingBinData(parentId: string): Promise<void> {
+		await this.tldrawDocumentApi.deleteByDocName(parentId);
+	}
```

Comment:
Config must be entered here and then X-API-KEY in the headers as the second parameter